file_formats/ie_formats/cre_v1.htm

```diff
@@ -415,63 +424,58 @@
           </tr>
           <tr>
             <td>0x0076</td>
-            <td>1 (byte)</td>
-            <td>(unused in BG2) Unknown proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
-          </tr>
-          <tr>
-            <td>0x0077</td>
-            <td>1 (byte)</td>
-            <td>(unused in BG2) Unknown proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
-          </tr>
-          <tr>
-            <td>0x0078</td>
-            <td>1 (byte)</td>
-            <td>(unused in BG2) Unknown proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
-          </tr>
-          <tr>
-            <td>0x0079</td>
-            <td>1 (byte)</td>
-            <td>(unused in BG2) Unknown proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
-          </tr>
-          <tr>
-            <td>0x007a</td>
-            <td>1 (byte)</td>
-            <td>(unused in BG2) Unknown proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>5 (bytes)</td>
+            <td><span class="unknown">Unused</span></td>

```

Comment:
Don't some mods use these extra proficiency stats? IOW it may be better to leave it as "unused proficiencies" rather than just unused. And the unknown tags are not needed. Same below.

Reply:
Are these extra proficiencies only used in BG1? In that case I would also update the field descriptions of offsets 0x6b .. 0x75.

Reply:
BG1-style come before, these 20 were just extra reservations.